inc/class-sitemaps-index.php

```diff
@@ -10,7 +10,7 @@
  * Class Core_Sitemaps_Index.
  * Builds the sitemap index page that lists the links to all of the sitemaps.
  */
-class Core_Sitemaps_Index extends Core_Sitemaps_Provider {
+class Core_Sitemaps_Index {
```

Comment:
I've specifically made this change because I think we need to maintain a distinction between the sitemap index and the individual sitemaps which will be listed in the index. There are definitely common patterns between them, but I wanted to decouple these from a conceptual point of view so our code can be more clear and keep a separation of concerns between the code that provides and index, and the code that provides a sitemap.